Code/Commando/DlgWOLSettings.cpp

```diff
@@ -454,12 +454,12 @@ void DlgWOLSettings::InitPersonaCombo(void)
 	if (combo)
 		{
 		// Populate the persona combo box with the list of WWOnline logins.
-		const LoginInfoList& logins = LoginInfo::GetList();
-		const unsigned int count = logins.size();
+			const LoginInfoList& logins = LoginInfo::GetList();
```

Comment:
Odd indentation changes. Probably due to the odd indentation in the file in the first place.